runtime/j9vm/jvm.c

```diff
@@ -1514,6 +1520,80 @@ static jint initializeReflectionGlobals(JNIEnv * env, BOOLEAN includeAccessors)
  *	DLL: jvm
  */
 jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args) {
+	return JNI_CreateJavaVM_impl(pvm, penv, vm_args, FALSE);
+}
+
+int32_t JNICALL JITServer_CreateServer(JITServer **jitServer, void *vm_args) {
+	JNIEnv *env = NULL;
```

Comment:
Coding style throughout: function return type and opening `{` should each be on separate lines:
```
jint JNICALL
JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args)
{
```